README.md

```diff
@@ -133,8 +133,6 @@ After installing tkn, create a link as kubectl-tkn
 $ ln -s /usr/local/bin/tkn /usr/local/bin/kubectl-tkn
   ```
 
-`kubectl` will find any binary named `kubectl-*` on your `PATH` and consider it as a plugin
-
   ```shell
 $ ./kubectl plugin list
 /usr/local/bin/kubectl-tkn
```

Comment:
Maybe separate this line from kubectl plugin list and say: 

You should see the following after running `kubectl plugin list` if `tkn` is available as a plugin:

```suggestion
/usr/local/bin/kubectl-tkn
```